spec/models/chunk_spec.rb

```diff
@@ -1,60 +1,82 @@
 require 'rails_helper'
 
 RSpec.describe Chunk do
-  subject { create(:chunk, content: "subject", embedding_content:) }
+  subject { create(:chunk, excerpt:, embedding_content:) }
 
   let(:document) { create(:document) }
-  let(:embedding_content) { nil }
+  let(:excerpt) { "subject" }
+  let(:embedding_content) { "surround subject sound" }
   let(:chunks) { document.chunks.sort }
 
-  before do
-    create(:chunk, document:)
-    create(:chunk, document:)
-    document.chunks << subject
-    create(:chunk, document:)
-    create(:chunk, document:)
+  describe "creating a chunk" do
```

Comment:
Not a big deal, but if you're looking for an arg to `describe` that's consistent with the pattern of naming after methods, you could use `create!`.

Reply:
Yes, thanks.